frontend/src/views/PurchasePage.vue

```diff
@@ -454,6 +456,12 @@ export default {
         response.json().then((jsonPurchase) => {
           this.originalPurchase = jsonPurchase
           this.purchase = JSON.parse(JSON.stringify(jsonPurchase))
+          if (this.duplicating) {
+            this.purchase.priceHt = null
+            this.purchase.date = null
+            this.purchase.invoiceFile = null
+            this.purchase.importSource = "Duplication"
+          }
```

Comment:
T'en penses quoi de setter `this.purchase.id = null` ici ? Ça ferait que `isNewPurchase` fonctionne tout seul et on n'aurait éventuellement pas besoin du changement de la ligne 350 : `id: this.duplicating ? null : this.purchase.id,`